BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -510,39 +534,48 @@ extension ConvivaAnalytics: BitmovinPlayerListenerDelegate {
 
     // MARK: - Ad events
     func onAdStarted(_ event: AdStartedEvent) {
+        var adInfo = event.ad.adInfo
         let adPosition: ConvivaSDK.AdPosition = AdEventUtil.parseAdPosition(
             event: event,
             contentDuration: player.duration
         )
-        var adAttributes = [String: Any]()
-        adAttributes["c3.ad.position"] = adPosition.rawValue
-        videoAnalytics.reportAdBreakStarted(
-            AdPlayer.ADPLAYER_CONTENT,
-            adType: AdTechnology.CLIENT_SIDE,
-            adBreakInfo: adAttributes
+        adInfo["c3.ad.position"] = adPosition.rawValue
+
+        adAnalytics.reportAdLoaded(adInfo)
+        adAnalytics.reportAdStarted(adInfo)
+        adAnalytics.reportAdMetric(
+            CIS_SSDK_PLAYBACK_METRIC_PLAYER_STATE,
+            value: PlayerState.CONVIVA_PLAYING.rawValue
         )
     }
 
     func onAdFinished() {
-        videoAnalytics.reportAdBreakEnded()
+        adAnalytics.reportAdEnded()
     }
 
     func onAdSkipped(_ event: AdSkippedEvent) {
-        customEvent(event: event)
```

Comment:
The web integration kept the `customEvents` reporting for now to not break existing integrations.

https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/pull/104#discussion_r1625646064

Reply:
https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva/pull/68/commits/1b42d71be7815a588413001c82e1ea94d74a5464
https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva/pull/68/commits/3de0a9aef58d8a4d022048ef779c6ab0ff481032